NuGet.Config

```diff
@@ -59,9 +59,11 @@
       <package pattern="envdte80" />
       <package pattern="envdte90" />
       <package pattern="envdte90a" />
+      <package pattern="Microsoft.Build.Framework" />
       <package pattern="Microsoft.DataAI.NuGetRecommender.Contracts" />
       <package pattern="microsoft.internal.visualstudio.*" />
       <package pattern="Microsoft.Internal.VisualStudio.Shell.Framework" />
+      <package pattern="Microsoft.NET.StringTools" />
```

Comment:
I recall there being concern about getting these items from the vside feed. 

`Microsoft.VisualStudio.Markdown.Platform` depends on a version of the SDK which is using the 17.13.x versions of both `Microsoft.NET.StringTools` and `Microsoft.Build.Framework`. Since these aren't released yet I would need to pull these from the VSIDE feed. 

I have another way of accomplishing this, where I created a new NuGet.config for the `NuGet.PackageManagement.UI` and set the PackageSourceMapping to * for the VSIDE source. It required a bunch of other changes

Reply:
One concern is that now by specifying the full id, you're mapping these packages to this feed only.
If we need something temporarily, I think we should double map instead since we own both feeds.
We should move away from it eventually when the versions are released. 

> I have another way of accomplishing this, where I created a new NuGet.config for the NuGet.PackageManagement.UI and set the PackageSourceMapping to * for the VSIDE source

This won't be enough since you need all the parents of NuGet.PackageManagement.UI to share the same config as well. 

It's a best practice to use a single config and everything that works with that. 
Final thing is that nested configs don't really work in VS.